using_images/s2i_images/dot_net_core.adoc

```diff
@@ -0,0 +1,91 @@
+[[using-images-using-dot-net-core]]
+= .NET Core
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[benefits-of-using-dot-net-core]]
+== Benefits of Using .NET Core
+
+link:http://developers.redhat.com/dotnet/[.NET Core] is a general purpose
+development platform featuring automatic memory management and modern
+programming languages. It allows users to build high quality applications
+efficiently. .NET Core is available on Red Hat Enterprise Linux (RHEL 7) and
+{product-title} via certified containers. .NET Core offers:
+
+* The ability to follow a microservices-based approach, where some components are
+built with .NET and others with Java, but all can run on a common, supported
+platform in Red Hat Enterprise Linux and {product-title}.
+* The capacity to more easily develop new .NET Core workloads on Windows;
+customers are able to deploy and run on either Red Hat Enterprise Linux or
+Windows Server.
+* A heterogeneous data center, where the underlying infrastructure is capable of
+running .NET applications without having to rely solely on Windows Server.
+* Access to many of the popular development frameworks such as .NET, Java, Ruby,
+and Python from within {product-title}.
+
+[[dot-net-core-supported-versions]]
+== Supported Versions
+
+* .NET Core version 1.0
+** Supported on Red Hat Enterprise Linux (RHEL) 7 {product-title} versions 3.3 and later
+
+ifdef::openshift-enterprise[]
+[[dot-net-core-installing-images]]
+== Images
+
+Image stream definitions for the .NET Core on RHEL S2I image are now added
+during {product-title} installations.
+endif::openshift-enterprise[]
+
+The RHEL 7 images are available through Red Hat's subscription registry using:
+
+----
+$ docker pull registry.access.redhat.com/dotnet/dotnetcore-10-rhel7
+----
+
+To use these images, you can either access them directly from these
+xref:../../architecture/infrastructure_components/image_registry.adoc#architecture-infrastructure-components-image-registry[image
+registries], or push them into your
+xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[{product-title}
+Docker registry]. Additionally, you can create an
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#image-streams[image
+stream] that points to the image, either in your Docker registry or at the
+external location. Your {product-title} resources can then reference the
+link:https://github.com/redhat-developer/s2i-dotnetcore/blob/master/dotnet_imagestreams.json[image stream definition].
+
+[[dot-net-core-configuration]]
+== Configuration
+
+The .NET Core image currently does not support any environment variables. No
+configuration is necessary.
+
+ifdef::openshift-enterprise[]
+[[dot-net-quickly-deploy-applications]]
+== Quickly Deploying Applications from .NET Core Source
+
+[IMPORTANT]
+====
+The .NET image stream must first be installed. If you ran a standard
```

Comment:
`.Net image stream` could be made a link to: https://github.com/redhat-developer/s2i-dotnetcore/blob/master/dotnet_imagestreams.json as well.